Doc/library/math.rst

```diff
@@ -366,17 +366,20 @@ Trigonometric functions
 
 .. function:: acos(x)
 
-   Return the arc cosine of *x*, in radians.
+   Return the arc cosine of *x*, in radians. The result is between ``-1.0`` and
```

Comment:
I think this should say: "between 0 and pi"

Reply:
Yes, sorry for the silly mistake. :(